src/System.Private.CoreLib/shared/System/Reflection/SignatureByRefType.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+#nullable enable
 
```

Comment:
Nit: remove the line in between namespace and `#nullable enable`. 

I see some other files with the same issue, will not comment on those, but please take care of all of them.

Reply:
sure